src/Styling/Project00.css

```diff
@@ -0,0 +1,44 @@
+/* Hero section */
+.Project00_hero_section {
```

Comment:
Is the “Project00” prefix really necessary? Your file is already called Project00.css . it’s safely assumed these styles are for the corresponding react file.

Reply:
removed the prefix